string_assignment.js

```diff
@@ -0,0 +1,66 @@
+//STRING EXERCISES
+
+
+// //1.
+// let myStr = 'You can do it!'
+//   console.log(myStr.charAt(0));
+
+
+// //2.
+// let myStr = 'I did it!'
+//   console.log(myStr.length);
+
+// //3.
+// let myStr = 'Oops I did it again'
+//   console.log(myStr.charAt(myStr.length-1));
+//
+// //4.
+
+// let myStr = 'deon'
+//   console.log(myStr.substring(0, myStr.length-1) + myStr[myStr.length-1].toUpperCase())
+
+//// Just practicing how to make the first character in a string uppercase.
+// let myStr = 'shayna'
+//   console.log(myStr[0].toUpperCase() + myStr.slice(1))
+
+
+
+// //5.
+//
+// for (let drEvil = 10; drEvil <= 1000000000; drEvil *= 10) {
+//     if (drEvil === 1000000000) {
+//       console.log(drEvil + ' dollars pinky')
+//   }else {
+//     console.log(drEvil + ' dollars')
+//   }
+// }
+
+
+
+// //6.
+// let verStr = 'swimming'
+// if (verStr.length < 3) {
+//   console.log(verStr)
+//     }else if (verStr.slice(-3) == 'ing') {
```

Comment:
One minor thing here is indentation. You will get better with JS syntax as you code more but indentation is important as it makes your code more readable to other folks. Just keep that in mind next time. Come ask me for any resources for proper indentation.